### ISSUE TITLE
Can't upgrade to Craft 3.7 with current install of Craft Reporter

### ISSUE THREAD

**Issue by RyanRoberts** [webmenedzser/craft-reporter#15]
Hey there, I'm getting the following message while attempting to upgrade to Craft 3.7.2:

```sh
Performing update with Composer ... error: An error occurred

Output:

Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - webmenedzser/craft-reporter is locked to version 1.12.0 and an update of this package was not requested.
    - webmenedzser/craft-reporter 1.12.0 requires craftcms/cms ~3.1.10 || ~3.2.10 || ~3.3.10 || ~3.4.10 || ~3.5.0 || ~3.6.1 -> found craftcms/cms[3.1.10, ..., 3.6.18] but it conflicts with your root composer.json require (3.7.2).
```

🤔

**Comment by Saboteur777** [comment]
Hey Ryan, thanks for opening this issue! I will release a new version in the coming days. Sorry for the delay!

**Comment by Saboteur777** [comment]
Just released 1.12.1 (https://github.com/webmenedzser/craft-reporter/compare/v1.12.0...v1.12.1) with a fix for this, thank you again for the heads-up!

**Comment by RyanRoberts** [comment]
Thanks 👍
